app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt

```diff
@@ -97,15 +97,15 @@ class HomeViewModel @Inject constructor(
 
     fun checkRequirements(onRequirement: (HomeRequirement) -> Unit) {
         viewModelScope.launch {
-            val userId = getSelf().first().id
+            val userId = observeSelf().first()
```

Comment:
suggestion: it's not only a `userId` anymore, maybe change the name to `user` or `selfUser`